TUTORIAL.md

```diff
@@ -951,6 +951,18 @@ input CollectionInput {
 *Rule #21: Structure mutation inputs to reduce duplication, even if this
  requires relaxing requiredness constraints on certain fields.*
 
+As in the example above, `collectionUpdate` takes two arguments. The first
+selects the collection to update, and the second provides the update data. An
+alternative to this would be one argument, `CollectionInput!`, which would
+include the ID. In fact, [some older Shopify APIs follow this
+pattern](https://shopify.dev/docs/api/admin-graphql/2023-07/mutations/productUpdate).
+However, this  is no longer recommended, as it makes it difficult to determine
+which parts of the call relate to 'select', and which relate to 'update'.
+
+*Rule #22: For update mutations, the parameter relating to selecting the entry
+must be separate to the parameter providing the change data. The parameter
+relating to selecting the entry must come first.*
```

Comment:
> The parameter relating to selecting the entry must come first.

Let's just omit this. Ordering schema elements in GraphQL isn't well defined and some implementations auto order alphabetically. If you're talking about writing a mutation operation here, again there's nothing to enforce an order.

Reply:
Should we use the term "argument" instead of "parameter" here?

Reply:
Oh, yeah in the generic sense 👍 that's fine

Reply:
Actually, we should use `argument` here because the rule is about schema _definitions_ not operations.